--- gnurobbo-0.68/gnurobbo/Makefile.orig	2023-09-22 21:11:40.226215359 +0200
+++ gnurobbo-0.68/gnurobbo/Makefile	2023-09-22 21:18:47.603900056 +0200
@@ -61,7 +61,7 @@ SDL_CONFIG?=sdl-config
 
 
 
-CFLAGS+=-d -Wall  `$(SDL_CONFIG) --cflags` -DPLATFORM_PC -DVERSION=\"$(VERSION)\" $(SMOOTH_SCRL) $(LIGHTNING)\
+CFLAGS+=$(OPTFLAGS) -Wall  `$(SDL_CONFIG) --cflags` -DPLATFORM_PC -DVERSION=\"$(VERSION)\" $(SMOOTH_SCRL) $(LIGHTNING)\
 	-DPACKAGE_DATA_DIR=\"$(PACKAGE_DATA_DIR)\" $(FONT_USE_PIXMAP) $(MUSIC) $(DESIGNER)
 LDFLAGS=
 # Use these instead for debugging and/or profiling (mainly intended for GNU Robbo developers)
