diff -urN gnurobbo-0.57-org/gnurobbo-0.57/game.c gnurobbo-0.57/gnurobbo-0.57/game.c
--- gnurobbo-0.57-org/gnurobbo-0.57/game.c	2002-11-15 23:42:36.000000000 +0100
+++ gnurobbo-0.57/gnurobbo-0.57/game.c	2005-02-25 15:23:29.324433184 +0100
@@ -21,7 +21,8 @@
  *
  */
 #include "game.h"
-#include "time.h"	/* for random data */
+#include <time.h>	/* for random data */
+#include <stdlib.h>
 
 /***********************************************************/
 /* This function checks if Robbo is killed - are there *****/
